src/test/java/org/openstreetmap/atlas/generator/AtlasGeneratorTest.java

```diff
@@ -0,0 +1,57 @@
+package org.openstreetmap.atlas.generator;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.geography.boundary.CountryBoundary;
+import org.openstreetmap.atlas.geography.boundary.CountryBoundaryMap;
+import org.openstreetmap.atlas.geography.sharding.DynamicTileSharding;
+import org.openstreetmap.atlas.geography.sharding.Shard;
+import org.openstreetmap.atlas.geography.sharding.Sharding;
+import org.openstreetmap.atlas.geography.sharding.SlippyTile;
+import org.openstreetmap.atlas.streaming.resource.InputStreamResource;
+
+/**
+ * @author matthieun
+ */
+public class AtlasGeneratorTest
+{
+    @Test
+    public void testRoughFilterShards()
+    {
+        final CountryBoundaryMap countryBoundaryMapHTI = new CountryBoundaryMap(
+                new InputStreamResource(
+                        () -> AtlasGeneratorTest.class.getResourceAsStream("HTI.txt")));
+        final CountryBoundaryMap countryBoundaryMapJAM = new CountryBoundaryMap(
+                new InputStreamResource(
+                        () -> AtlasGeneratorTest.class.getResourceAsStream("JAM.txt")));
+
+        final List<CountryBoundary> countryBoundaryHTI = countryBoundaryMapHTI
+                .countryBoundary("HTI");
+        final List<CountryBoundary> countryBoundaryJAM = countryBoundaryMapJAM
+                .countryBoundary("JAM");
+
+        final Sharding sharding = new DynamicTileSharding(new InputStreamResource(
+                () -> AtlasGenerator.class.getResourceAsStream("tree-6-14-100000.txt")));
```

Comment:
The sharding tree already exists in src/test/resources/org/openstreetmap/atlas/geography/boundary/tree-6-14-100000.txt.gz - let's reuse that!

Reply:
So that is in another project (atlas), and unfortunately the test resources are not packaged in the library jar I don't think...

Reply:
ohhh good point! Forgot that this is atlas-generator, not atlas - sorry!